main.tex

```diff
@@ -18,6 +18,33 @@
 \begin{document}
 
 \maketitle
+\subsubsection {Facebook}
+Eine Studie von \textcite{empowermentThroughFacebook}, welche zwischen 2011 und 2012 in New Delhi durchgeführt wurde, zeigt, dass Facebook zu der Zeit eine große Rolle in der Gesellschaftsentwicklung spielte. 
+\medskip
+Da zu 2012 eine Regulierung von bis zu 200 SMS pro Tag veröffentlicht wurde(Press Information Bureau,
+Government of India, 2012), stieg das Interesse von Facebook zu der Zeit stark an,
+da Handybesitzer weiterhin unlimitiert mit ihren Freunden kommunizieren wollten.
+Durch das Kennenlernen und Nutzen von Facebook bildete sich eine Gesellschaft, welche bislang der Globalisierung
+durch das Internet ausgeschlossen waren und nun erstmals eine transnationale Identität erstellen konnte.
+\medskip
+Bis 2015 wuchs die Nutzeranzahl von Facebook in Indien geradezu explosionsartig, und mit derzeit 135 Millionen Nutzer waren gerademal 25\% weiblich\autocite{slideshareIndia}. Das liegt vorallem daran, dass vorallem jungen indischen Frauen, aus elterlichen Prinzipien, 
+wenig Freiheit gelassen wurde sich mit Freunden zu treffen, oder gar Technologie
+zu besitzen. Dieser Anteil sollte sich laut Aktivisten in der Zukunft stark ändern, 
```

Comment:
`Technologie
zu besitzen` klingt komisch :)

Reply:
Die Kulturellen Aspekte dessen, dass Frauen nicht in Internet dürfen (und noch ne Menge anderes) ist eine der Eigenheiten Indiens und kann gern noch hervorgehoben werden.

Auszug aus `everdayLife`, s 10f:

```
Another important claim was that the Freebasics would help women access information. 
This could only be true for those in the lower socio-economic classes as women belonging 
to upper socio-economic classes, already access the internet and social media sites. However, 
in the lower middle classes and lower socio-economic classes, caste issues and strict social 
surveillance of young unmarried women often makes it difficult for them to access even 
phones, let alone use the internet or social media.  There is a prevalent social notion that 
access to phones might endanger a woman’s chastity.41 Unmarried young women with 
school education have the highest potential to access the internet of all the people, but are 
cut off from tools to gain such access. Once married they may gain the right to own a 
phone, yet access to the internet might still be guarded by their in-laws. 
For example: Rajah is a 22 year old studying Computer Science. He is always signed into 
Facebook through his smart phone, however, when it comes to his sister who studies in the 
same college as he does, Rajah exerts a control on her and doesn’t allow her to access social 
media of any kind. His reason for keeping out his sister from the ‘clutches’ of social media is 
media and the internet as a masculine space42 dangerous for women, occur often. Strict 
surveillance over boundaries of time and space of access make everything else dangerous and 
```

Reply:
gehe das eben durch, streiche die irrelevanten Sachen und adde das hier. :)